std_msgs/Header header
int32 target_x
int32 target_y
float32 psuedo_distance
